crates/common_enums/src/enums.rs

```diff
@@ -2258,6 +2258,14 @@ pub enum PaymentSource {
     ExternalAuthenticator,
 }
 
+#[derive(Default, Debug, Clone, serde::Deserialize, serde::Serialize, strum::EnumString)]
```

Comment:
Strum::EnumString needed?

Reply:
Yes, we need to add strum::EnumString to our enum definition. It implements FromStr which is required for parsing.